engine/class_modules/sc_death_knight.cpp

```diff
@@ -5947,6 +5957,11 @@ struct remorseless_winter_damage_t : public death_knight_spell_t
 
     m *= 1.0 + p() -> buffs.gathering_storm -> stack_value();
 
+    if ( p() -> legendary.biting_cold.ok() )
```

Comment:
You can even apply the modifier directly on base_multiplier during construction.